### DISCUSSION TITLE
No response from GitHub support over 2 weeks

### DISCUSSION THREAD

**Discussion by EricZil** [community/community#58310] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hey,

How long does it take GitHub Support to respond? My ticket was opened over two weeks ago and has yet to get a response. 
--
Thank you!
Eric

**Comment by the-officialjosh** [comment]
If it has been over two weeks and you haven't received a response, it's advisable to follow up on your ticket. You can reply to the initial email you received when opening the ticket or reach out to GitHub Support through their contact channels, such as the GitHub Support website or their official support Twitter account.

When reaching out, make sure to provide your ticket number or any relevant details to help them locate and address your inquiry promptly.

**Comment by baharbaba** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Reply by LiteBrite82** [reply]
Hi @baharbaba 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Reply by baharbaba** [reply]
i did get an response from github before just by being a karen but they didnt solved my problem they didnt even returned me after that response and im just trying to get attention.
